cocktails/easy/wikipedia_pina_colada.md

```diff
@@ -0,0 +1,15 @@
+# Textbook (Wikipedia inspired) Piña Colada
+
+## Ingredients
+
+* 1 part white rum
```

Comment:
can maybe specify a guide unit

Reply:
such as 40ml/40ml/120ml rum/coconut/juice? would fill a glass

Reply:
I think parts is fine. Many cocktail recipes use parts and it's ran easy way to estimate how much to use for any given glass, like ratios.